source/auth/auth.rst

```diff
@@ -1158,6 +1163,9 @@ Q: Why does SCRAM sometimes SASLprep and sometimes not?
 Version History
 ===============
 
+Version 1.9.2 Changes
```

Comment:
As above.  It's worth a minor version bump

Reply:
1.10.0

Reply:
Updated.